workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/customer-portal/components/ActivationStatus/LiferayExperienceCloud/components/SetupLXCModal/components/SetupLXCForm/components/SetupLXCPage/hooks/useSubmitLXCEnvironment.js

```diff
@@ -226,8 +219,8 @@ export default function useSubmitLXCEnvironment(
 				await updateRaysourceContact(
 					removeContactRoleRaysource,
 					removeHighPriorityContactList,
+					oAuthToken,
```

Comment:
@jennyhchen I am not sure what Bchan will say about these, they arent alphabetized, but they werent before, its probably fine, but also probably didnt need to switch up the order. But we can send it and see what he says